practice_06.08.2019_System.IO/Task2_FileCreation/FileCreation.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.IO;
+
+
+//Разработайте приложение, которое позволяет создавать текстовый файл по пути C:\Temp\userText.txt и записывать в него текст,
+//полученный от пользователя.
+
+namespace practice_06._08._2019_System.IO.Task2_FileCreation
+{
+    public class FileCreation
+    {
+        public string tempFolder = @"D:\Temp\";
+
+        public string textFile = "userText.txt";
+
+        public string CreateFile()
+        {
+            try
+            {
+                if (Directory.Exists(this.tempFolder))
+                {
+                    Console.WriteLine($"{Path.GetDirectoryName(this.tempFolder)} folder is already exist");
+                    WriteText(this.tempFolder, this.textFile);
+                }
+                else
+                {
+                    DirectoryInfo newFolder = Directory.CreateDirectory(this.tempFolder);
+                    Console.WriteLine($"{newFolder.Name} folder was created");
+                    WriteText(this.tempFolder, this.textFile);
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+            }
+
+            return this.tempFolder + this.textFile;
+        }
+
+        public string WriteText(string path, string textFile)
+        {
+            if (File.Exists(path + textFile))
+                File.Delete(path + textFile);
+           
+            Console.WriteLine("Write some text:");
+            var inputText = Console.ReadLine();
+
+            var writer = new StreamWriter(path + textFile, true);
```

Comment:
We did not learn streams but if you use them, please make sure that all resources will be collected when code is completed.
Please use Dispose() or use () { } statement.

Reply:
I used writer.Close(). Is it not the same as Dispose()? I'm not sure that I understand how to use it.
Skipped